scripts/migrations/2020-08-update_migration_created_date.py

```diff
@@ -0,0 +1,118 @@
+    except KeyError:
+        try:
+            bucket = item["payload"]["bucket"]
+            key = item["payload"]["key"]
+        except:
+            record_error(
+                item["id"], item["version"], f"Cannot find s3 bucket, key in {item}"
+            )
+        else:
+            return bucket, key
+    else:
+        return bucket, key
+
+
+def get_vhs_json(id, version, bucket, key):
+    try:
+        return json.loads(s3.Object(bucket, key).get()["Body"].read().decode("utf-8"))
+    except ClientError as e:
+        record_error(id, version, f"Cannot read s3 entry for {id}: {item}: {e}")
+
+
+def put_vhs_json(id, version, bucket, item, content):
+    filename = f"{uuid.uuid4()}.json"
+    key = f"{id}/{version}/{filename}"
+    item["payload"]["bucket"] = bucket
+    item["payload"]["key"] = key
+    try:
+        s3.Object(bucket, key).put(Body=(bytes(json.dumps(content).encode("UTF-8"))))
+        dynamodb.Table(vhs_table).put_item(Item=item)
+    except ClientError as e:
+        record_error(
+            id,
+            version,
+            f"Error updating backfill vhs object {id}/{version} to s3://{bucket}/{key}: {e}",
+        )
+
+
+def get_backfill_item(id, version):
+    try:
+        return dynamodb.Table(backfill_vhs_table).get_item(
+            Key={"id": id, "version": version}
+        )["Item"]
+    except KeyError:
+        record_error(id, version, f"Cannot find backfill storage manifest!!!!")
+
+
+def is_expected_diff(id, version, current_json, backfill_json):
+    diff = DeepDiff(current_json, backfill_json, ignore_order=True)
+
+    if diff:
+        values_changed = diff.pop("values_changed", None)
+        items_added = diff.pop("iterable_item_added", None)
+        if values_changed and values_changed.keys() - ["root['createdDate']"]:
+            record_error(id, version, f"Unexpected values changed in {values_changed}!")
+            return False
+        if items_added and items_added.keys() - ["root['replicaLocations'][1]"]:
+            record_error(id, version, f"Unexpected values changed in {items_added}!")
+            return False
+        if diff:
+            record_error(id, version, f"Unexpected values changed in {diff}!")
+            return False
+    return True
+
+
+for item in scan_table(TableName=vhs_table):
+    id = item["id"]
+    version = item["version"]
+    bucket, key = get_bucket_key(item)
+    if bucket and key:
+        vhs_content = get_vhs_json(id, version, bucket, key)
+        if vhs_content:
+            created_date = vhs_content["createdDate"]
+            backfilled_item = get_backfill_item(id, version)
+            if backfilled_item:
+                backfilled_bucket = backfilled_item["payload"]["bucket"]
+                backfilled_key = backfilled_item["payload"]["key"]
+                backfilled_json = get_vhs_json(
+                    id, version, backfilled_bucket, backfilled_key
+                )
+                if backfilled_json:
```

Comment:
Might be worth a line here:

```python
if backfilled_json == vhs_content:
    continue
```

so you won't keep writing the same manifests over and over.